groups/live-windows-security-groups/profiles/heritage-live-eu-west-2/vars

```diff
@@ -28,4 +32,146 @@ test_dev_generate_135_cidr_block = [
   "172.16.101.82/32",
   "172.16.202.91/32",
   "172.16.200.200/32"
+]
+
+# ------------------------------------------------------------------------------
+# Smart Vault 1 Server Security Group Variables
+# ------------------------------------------------------------------------------
```

Comment:
I would make these locals instead of vars, its a single environment so we dont have to worry about different inputs and its less work because you dont need to declare variables for each one.

Reply:
Changed to locals and vars removed.